Dockerfile

```diff
@@ -42,5 +42,4 @@ LABEL org.opencontainers.image.authors="Author"
       org.example.image.alpineversion="3.9"                                             \
       org.example.image.schemaversion="1.0"
 
-ENTRYPOINT [ "trivy" ]
-CMD [ "--help" ]
\ No newline at end of file
+CMD [ "trivy" ]
```

Comment:
you are missing a new line here

Reply:
This is weird. See screenshot:

![Screen Shot 2019-07-29 at 9 12 32 PM](https://user-images.githubusercontent.com/6694151/62093013-9e4f5e80-b245-11e9-8efb-431b585825b8.jpg)

Reply:
What's missing?

Reply:
The final new line. You always add a final new line to files in the linux world.

Reply:
Humm didn't know it. Cheers!